README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_tree-from-object-DOM/)
+    - [DEMO LINK](https://pushkalov.github.io/js_tree-from-object-DOM/)
```

Comment:
<img width="1398" alt="image" src="https://user-images.githubusercontent.com/38065883/175119363-20adaf2e-51cb-4b37-8070-7472c02b3cfa.png">
Demo page is empty